### DISCUSSION TITLE
Transfer rules from fluent to vector

### DISCUSSION THREAD

**Discussion by KaMaToZzz** [vectordotdev/vector#12200] [Q&A] [answered]
Hello,

I have some rules from Fluetn taking from docker swarm dotnet apps and its look like:

`https://pastebin.com/JZgSL4gE` 

Can some one help change this rewrite rules to vector?

**Comment by jszwedko** [comment, marked as answer]
Hi @KaMaToZzz ,

Looking at this configuration, you'd probably find the `remap` and `filter` Vector transforms helpful. I'd recommend starting there, but you can let us know if you have any issues. Notably Vector's `elasticsearch` sink doesn't yet support multiple hosts #3649
